integration/raft/config_test.go

```diff
@@ -814,29 +832,36 @@ var _ = Describe("EndToEnd reconfiguration and onboarding", func() {
 			o4process := ifrit.Invoke(orderer4Runner)
 			Eventually(o4process.Ready(), network.EventuallyTimeout).Should(BeClosed())
 			ordererProcesses = append(ordererProcesses, o4process)
+			// Get the last config block of the testchannel
+			configBlock := nwo.GetConfigBlock(network, peer, o1, "testchannel")
+
+			By("Joining the fourth orderer to the channel, as consenter")
+			expectedChannelInfo := channelparticipation.ChannelInfo{
+				Name:              "testchannel",
+				URL:               "/participation/v1/channels/testchannel",
+				Status:            "onboarding",
+				ConsensusRelation: "consenter",
+				Height:            0,
+			}
+			channelparticipation.Join(network, o4, "testchannel", configBlock, expectedChannelInfo)
 
 			By("And waiting for it to sync with the rest of the orderers")
 			assertBlockReception(map[string]int{
-				"systemchannel": 10,
-				"testchannel":   8,
+				"testchannel": 8,
 			}, orderers, peer, network)
 
 			By("Ensuring orderer4 doesn't serve testchannel2 and testchannel3")
 			env = CreateBroadcastEnvelope(network, peer, "testchannel2", []byte("hello"))
 			resp, err = ordererclient.Broadcast(network, o4, env)
 			Expect(err).NotTo(HaveOccurred())
-			Expect(resp.Status).To(Equal(common.Status_SERVICE_UNAVAILABLE))
+			Expect(resp.Status).To(Equal(common.Status_BAD_REQUEST))
+			Expect(orderer4Runner.Err()).To(gbytes.Say("channel creation request not allowed because the orderer system channel is not defined"))
```

Comment:
This error message is misleading. Are we planning to update this message later?

Reply:
Good point. Yes. When system channel support will be removed from the registrar we'll need to change this response.